modules/nfd-rules-about.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/psap-node-feature-discovery-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nfd-rules-about_{context}"]
+= About the NodeFeatureRule custom resource
+
+`NodeFeatureRule` objects provide a method to create vendor- or application-specific labels and taints. It uses a flexible rule-based mechanism for creating labels and optionally taints based on node features.
+
+`NodeFeatureRule` objects are a `NodeFeatureDiscovery` custom resource designed for rule-based custom labeling of nodes. Some use cases include application-specific labeling or distribution by hardware vendors to create specific labels for their devices.
```

Comment:
I _think_ I like this sentence first in the about section because it defines CR. The other sentences in this cluster then go on to further describe the CR or modify that original description. I wonder if the paragraphs could be switched because the second gives the reader the "need to know" while the first gives the reader the sort of "in the weeds" details. Just a suggestion as always take it as you will.